### ISSUE TITLE
Accessibility: Improvements and suggestions

### ISSUE THREAD

**Issue by pocketcasts** [Automattic/pocket-casts-ios#30]
I identified several areas in the app where accessibility can be improved. I put more details in this google doc, which you should have access to with this [link](https://docs.google.com/document/d/1Lo1GbzK4xPWvlN2yvYriCjY-OaYF-LQ6lheIyI3DPpY/edit).

- [ ] Dynamic Text Size needs to be supported throughout the app, the only place I saw support for it was in the navigation bar.
- [ ] When reading over a podcast episode with VoiceOver it reads the `m` in `2m` as meters instead of minutes. I believe this may be fixed when I run it from Xcode but want to make sure it ships this way. Also, suggest changing the `m` to min so it is clearer to say that it's minutes instead of meters.

![Meters](https://user-images.githubusercontent.com/36495451/154548487-dc80ba55-5aaf-4176-9e7f-c8d581d50899.png)
 
- [x] On the Discover Tab the plus button for adding a subscription is shown as “Discover add”, which sounds like an internal name for the button. This should say subscribe like the plus button like the other podcasts that are being shown on this page.

![Discover Add](https://user-images.githubusercontent.com/36495451/154548685-6706501a-172d-426e-bbb5-7d2cd1daede3.png)

**Comment by sophiegyo** [comment]
Hello! For accessibility stuff, we had a question from a user about adding font size adjustability in iOS:
https://twitter.com/anishvarsha1/status/1615201889465012224

>When can we expect the accesibility settings for the iOS to get the system level font size or at the least a custom setting for the application. Would be a great add for getting the suite of features you app offers.

**Comment by joashrajin** [comment]
> My issue is that I have trouble navigating the discover section after doing a search. To be more precise: I can't reliably click the search result I want when I am searching for new podcasts. When VoiceOver reads the desired podcast, and I do a double tap with one finger in order to open it, nothing happens. 

6625894-zd-a8c

**Comment by joashrajin** [comment]
> Voiceover is the screen reader for blind people using iPhones, iPads, and Macs. Yesterday I noticed podcasts were not reading in the proper order inside my Unplayed filter. I also could not bring up the Player to get to the screen showing the player controls. Normally, the podcasts display in a sequential order with the rewind, pause, and fast forward buttons labeled by default before I even open the player screen. After examining the screen and Voiceover a bit more, I noticed that the screen recognition had been enabled for Pocket Casts automatically. This happens when Voiceover determines the regular interface for the application cannot be read without processing images on the screen and reformatting them in a way I can read. This is only necessary if the application has always been, or has become, inaccessible. Sadly, recent changes seem to have made the filter screen not accessible, so Voiceover tries to reconfigure the interface. This means words and text are reordered, the interface is confusing and unclear, and I can’t find or open the Player button as I once did. A button with such a label simply isn’t there anymore.  This was NOT how Pocket Casts usually works.
It would be wonderful if you please look into the filters screen of podcasts and ensure it works with Voiceover again.

 9361807-zd-a8c
